packages/code/src/Code.tsx

```diff
@@ -382,61 +352,48 @@ function Code({
     </div>
   );
 
-  const onScroll: React.UIEventHandler<HTMLDivElement | HTMLPreElement> = e => {
-    e.persist();
-    debounceScroll(e);
-  };
+  return (
+    <div className={wrapperStyle}>
+      {showWindowChrome && (
+        <WindowChrome chromeTitle={chromeTitle} darkMode={darkMode} />
+      )}
 
-  const commonWrapperProps = {
-    chromeTitle,
-    darkMode,
-    showWindowChrome,
-  } as const;
-
-  if (!multiline) {
-    return (
-      <CodeOuterWrapper {...commonWrapperProps}>
-        <div
-          {...(rest as DetailedElementProps<HTMLDivElement>)}
+      <div
+        className={css`
+          display: flex;
+        `}
+      >
+        <pre
+          {...(rest as DetailedElementProps<HTMLPreElement>)}
           className={wrapperClassName}
           onScroll={onScroll}
-          ref={scrollableSingleLine}
+          ref={scrollableElement}
         >
           {renderedSyntaxComponent}
-        </div>
+        </pre>
 
         {copyBar}
-      </CodeOuterWrapper>
-    );
-  }
-
-  return (
-    <CodeOuterWrapper {...commonWrapperProps}>
-      <pre
-        {...(rest as DetailedElementProps<HTMLPreElement>)}
-        className={wrapperClassName}
-        onScroll={onScroll}
-        ref={scrollableMultiLine}
-      >
-        {renderedSyntaxComponent}
-      </pre>
-
-      {copyBar}
-    </CodeOuterWrapper>
+      </div>
+    </div>
   );
 }
 
 Code.displayName = 'Code';
 
 Code.propTypes = {
   children: PropTypes.string.isRequired,
-  multiline: PropTypes.bool,
   language: PropTypes.oneOf(Object.values(Language)),
   darkMode: PropTypes.bool,
   className: PropTypes.string,
   showLineNumbers: PropTypes.bool,
   showWindowChrome: PropTypes.bool,
   chromeTitle: PropTypes.string,
+  highlightLines: PropTypes.arrayOf(
+    PropTypes.oneOfType([
+      PropTypes.arrayOf(PropTypes.number),
```

Comment:
I think a way to only accept a pair of numbers is 
```tsx
PropTypes.shape({ 0: PropTypes.number, 1: PropTypes.number })
```

Reply:
Ahh, smart.